src/backend/src/utils/projects.utils.ts

```diff
@@ -99,7 +100,7 @@ export const projectTransformer = (
     wbsNum,
     dateCreated: wbsElement.dateCreated,
     name: wbsElement.name,
-    status: wbsElement.status as WbsElementStatus,
+    status: calculateProjectStatus(project),
```

Comment:
line 135 of this file should also use this function

Reply:
wait actually that's a bug, that line should refer to the workpackage's status but it's referring to the project's instead